### PR TITLE
Refactor stern/tail.go

### PR DESCRIPTION
**Pull request by superbrothers**
- Use corev1client instead all of clientset
- Remove an useless argument from `tail.Start`
- Change `tail.Start` to return an error

### CODE REVIEW

File: stern/tail.go [hunk 1]
```diff
@@ -43,6 +45,9 @@ type Tail struct {
 	containerColor *color.Color
 	tmpl           *template.Template
 	active         bool
+
```
Comment (floryut) [inline stern/tail.go:48]:
```suggestion

```

File: stern/tail.go [hunk 2]
```diff
@@ -89,6 +95,9 @@ func NewTail(nodeName, namespace, podName, containerName string, tmpl *template.
 		closed:        make(chan struct{}),
 		tmpl:          tmpl,
 		active:        true,
+
```
Comment (floryut) [inline stern/tail.go:98]:
```suggestion

```

**Review by floryut — APPROVED** [review]
Neat ! 👍

**Comment by superbrothers** [comment]
Thank you for your review, and happy New Year! 🥳

**Comment by floryut** [comment]
> Thank you for your review, and happy New Year! 🥳

Hehe thank you man, happy new year to you too! Wishing all the best

**Pull request merged**